docs/api.rst

```diff
@@ -1442,6 +1442,71 @@ Voice
     :param after: The voice state after the changes.
     :type after: :class:`VoiceState`
 
+Polls
```

Comment:
These events are alphabetically sorted, P comes before V not after. The English alphabet is `ABCDEFGHIJLKMNOPQRSTUVWXYZ`